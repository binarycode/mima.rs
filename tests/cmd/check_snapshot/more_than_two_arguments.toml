args = 'check-snapshot one two three'
status.code = 2
