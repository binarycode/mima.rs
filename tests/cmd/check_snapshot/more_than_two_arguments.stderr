error: unexpected argument 'three' found

Usage: mima check-snapshot <GUEST_ID> <SNAPSHOT_ID>

For more information, try '--help'.
